src/forms/Slushbucket.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { Component } from "react";
+
+import MDSpinner from "react-md-spinner";
+import Scrollbars from "react-custom-scrollbars";
+import ClearButton from "../components/ClearButton.js";
+
+import styles from "./SlushBucket.css";
+import formStyles from "../styles/Forms.css";
+import inputStyles from "../styles/Input.css";
+import displayStyles from "../styles/Display.css";
+
+type choiceT = {value: string | number, display: string};
+export type choicesT = [choiceT];
```

Comment:
Dit is volgens mij een array met 1 element, dat een choiceT moet zijn. Een array van choiceTs is choiceT[].